lib/auth/apiserver.go

```diff
@@ -751,19 +751,41 @@ func (s *APIServer) u2fSignRequest(auth ClientI, w http.ResponseWriter, r *http.
 	return u2fSignReq, nil
 }
 
-type createWebSessionReq struct {
-	PrevSessionID   string `json:"prev_session_id"`
+// CreateWebSessionReq defines fields for requesting to create new web session.
+type CreateWebSessionReq struct {
+	// PrevSessionID is the id of a valid existing session.
+	PrevSessionID string `json:"prev_session_id"`
+	// AccessRequestID is an optional field that holds the id of an approved access request.
 	AccessRequestID string `json:"access_request_id"`
+	// User is the user name associated with the session id.
+	User string `json:"user"`
+	// SwitchBackDefaults is an optional field that defines default user roles and expiration values.
+	// These are the preserved values from first session creation after web login.
+	SwitchBackDefaults *WebSession `json:"session"`
+}
+
+// WebSession defines fields describing a session expiration and roles assigned to it.
```

Comment:
`WebSession` is a pretty generic name. How does this differ from the WebSession from the API types package? As this is used to represent the attributes of the original web session used to switch user back to their defaults, I would try to keep this meaning as part of the type.